library/openssl/fixed_length_secure_compare_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../../spec_helper'
+require 'openssl'
+
+describe "OpenSSL.fixed_length_secure_compare" do
+  it "returns true for two strings with the same content" do
+    input1 = "the quick brown fox jumps over the lazy dog"
+    input2 = "the quick brown fox jumps over the lazy dog"
+    OpenSSL.fixed_length_secure_compare(input1, input2).should be_true
+  end
+
+  it "returns false for two strings of equal size with different content" do
+    input1 = "the quick brown fox jumps over the lazy dog"
+    input2 = "the lazy dog jumps over the quick brown fox"
+    OpenSSL.fixed_length_secure_compare(input1, input2).should be_false
+  end
```

Comment:
**suggestion:** it also makes sense to illustrate that length is bytes count, not characters count (as far as I see in the implementation)